Lib/importlib/_bootstrap_external.py

```diff
@@ -1256,6 +1262,21 @@ def set_data(self, path, data, *, _mode=0o666):
                                         exc)
 
 
+def _exclude_directory_from_backups(directory):
+    """Exclude the directory from backups using a CACHEDIR.TAG file.
+
+    If the file exists or there's a permission error this is a no-op.
+    """
+    try:
+        with _io.FileIO(_path_join(directory, 'CACHEDIR.TAG'), 'x') as f:
+            f.write(b"""Signature: 8a477f597d28d172789f06886806bc55
+# This file is a cache directory tag automatically created by CPython.
```

Comment:
Not that it particularly matters, but I was curious why this doesn't match the comment given on https://bford.info/cachedir/. The "automatically" seems to be from mypy. Compare the following searches:

https://github.com/search?type=code&q=%22This+file+is+a+cache+directory+tag+created+by%22
https://github.com/search?type=code&q=%22This+file+is+a+cache+directory+tag+automatically+created+by%22

```suggestion
# This file is a cache directory tag created by CPython.
```

Reply:
I'm curious too but I can't remember now. Fair point.